packages/tools/devtools/devtools/api-report/devtools.alpha.api.md

```diff
@@ -32,6 +32,4 @@ export { IDevtoolsLogger }
 // @beta
 export function initializeDevtools(props: DevtoolsProps): IDevtools;
 
-// (No @packageDocumentation comment for this package)
```

Comment:
this delta seems. odd. There was a package documentation comment, so this shouldn't exist, but why was it here before your change? Did somone not update the API report and CI not catch that somehow?

Reply:
But the devtools package _does_ have a `@packageDocumentation` comment.
https://github.com/microsoft/FluidFramework/blob/main/packages/tools/devtools/devtools/src/index.ts#L18

Reply:
Exactly - don't know why the report said it didn't prior to this change.

Reply:
🤦‍♂️ Parsed the diff backwards.